### ISSUE TITLE
Max Quality Option

### ISSUE THREAD

**Issue by Revan335** [meeb/tubesync#286]
Hello,

a simple Max Quality option would be great to always get the best quality without having to know anything about the individual options and then be able to make a sensible decision.

This is a extension: https://github.com/meeb/tubesync/issues/285

Thanks a lot!

Greetings

**Comment by meeb** [comment]
By best quality I assume you mean just best resolution available? Likely this is going to be quite subjective, and once you get into 4k or higher is going to likely be restricted to just one codec. The defaults (I believe 1080p VP9 at the moment) are probably the most sensible out of the box. Do you imagine this to also consider codec, bitrate, HDR and high frame rate into a decision on what is "max quality"?

**Comment by Revan335** [comment]
> By best quality I assume you mean just best resolution available? Likely this is going to be quite subjective, and once you get into 4k or higher is going to likely be restricted to just one codec. The defaults (I believe 1080p VP9 at the moment) are probably the most sensible out of the box. Do you imagine this to also consider codec, bitrate, HDR and high frame rate into a decision on what is "max quality"?

Yes, the best/hightest available combine of resolution, codec, bitrate, HDR, HFR and Co.

The maximum quality option can be a separate point. And an extended point for individual settings like now, for the experts/advanced users.
This way you don't lose any features, but you don't have to worry about which of the options is the right one for you, if you don't have any expertise about it, but just want to load the best possible quality.

You can do it this way.
Point One Do you want the best/highest possible combination of resolution and codec? Point Two Do you also want to have bitrate, HDR, HFR .... or should this be included?
Or Separate Points for every Option Resolution, Codec, Bitrate, HDR, HFR ....

**Comment by tcely** [comment]
See also:
- #1344 
- #1117 
- #88
